### ISSUE TITLE
iscsi configuration how to

### ISSUE THREAD

**Issue by rburgst** [rancher/os#1967]
I am unsure on how to properly configure `iscsi`. I managed to get it to work manually by following the instructions at

https://github.com/rancher/os-services/pull/110

However, this fails to mention how you can configure iscsi so that a volume is mounted automatically at startup. 
Also I failed to get authentication to work, my guess is that `/etc/iscsi/iscscid.conf` might be the wrong place to configure auth credentials.

In short a proper documentation would be highly appreciated.

**RancherOS Version: (ros os version)** 1.0.3

**Where are you running RancherOS? (docker-machine, AWS, GCE, baremetal, etc.)**

`docker-machine`

**Comment by niusmallnan** [comment]
track this new issue #2636
